frontend/src/communications/userApi.js

```diff
@@ -14,8 +19,17 @@ export const login = (value) => {
   return api.post('/login', value)
 }
 
-export const signUp = async (value) => {
-  return await api.post('/registration')
+export const signUp = async (value, type) => {
+  const data = {
+    full_name: value.name,
+    tax_number: value.taxNumber,
+    address: value.address,
+    email: value.email,
+    type: type,
+    password: value.password,
+  }
+  console.log('data: ' + JSON.stringify(data))
```

Comment:
Vegyük ki a logokat, így lehet még a data változó is eltűnhet és rögtön a hívás helyén hozhatjuk létre a paramétereket. HTTP request debuggolására használjuk a megfelelő eszközöket, például Chrome Developer Tools Network füle.